sdv/metadata/errors.py

```diff
@@ -5,5 +5,9 @@ class MetadataError(Exception):
     """Error to raise when Metadata is not valid."""
 
 
+class InvalidMetadataError(Exception):
```

Comment:
Why create this new error? Couldn't we just use the `MetadataError` defined above?